Dockerfile.15sp3

```diff
@@ -23,7 +23,7 @@ COPY integration/create-oscrc.sh /tmp/connect/integration/create-oscrc.sh
 RUN sh /tmp/connect/integration/create-oscrc.sh
 
 RUN echo 'gem: --no-ri --no-rdoc' > /etc/gemrc && \
-    gem install bundler --no-document
+    gem install bundler --no-document -v 2.3.27
```

Comment:
Is this change not needed on the other dockerfiles?

Reply:
added to 15SP0, too now. In the sle12 Dockerfiles this is already covered.